src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -314,11 +314,13 @@ class WorkflowManagerActor(backend: Backend) extends LoggingFSM[WorkflowManagerS
       BackendCallKey(descriptor.namespace.workflow.findCallByName(callName).get, key.index, key.attempt)
     def backendCallFromKey(descriptor: WorkflowDescriptor, callName: String, key: ExecutionDatabaseKey) =
       backend.bindCall(descriptor, callKey(descriptor, callName, key))
+
+    import cromwell.engine.finalcall.FinalCall._
```

Comment:
If it's an enhancing implicit (I'm assuming the .isFinalCall) and not an implicit implicit (magic auto-conversions) I don't see think it's a big deal. You'd only be using it if you intended to. If it is indeed a magic auto-conversion we should probably not have that :)